webpack.config.js

```diff
@@ -2,6 +2,7 @@
 const path = require("path");
 const webpack = require("webpack");
 const pkg = require("./package.json");
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
this can be added to `module.exports.plugins` and set `module.exports.mode = "development"` to see relative module sizes